lib/crowbar/client/app/upgrade.rb

```diff
@@ -367,6 +367,24 @@ def database(mode)
           catch_errors(e)
         end
 
+        desc "mode UPGRADEMODE",
+          "Set/Get the upgrade mode (normal|non_disruptive)"
+
+        long_desc <<-LONGDESC
+          `mode` allows you to switch the upgrade mode between `non_disruptive` and `normal`.
+          Calling it without any arguments will return the currently selected upgrade mode.
```

Comment:
It may be useful to write about the time when this can (or more likely cannot) be called.